README.md

```diff
@@ -61,6 +60,20 @@ podman build \
 You will want to push this image to a registry so that it can be used as an
 `osImageUrl` in a MachineConfig as documented in the OpenShift docs.
 
+If you're using Quay.io, start by logging into your account with Podman:
+
+```
+podman login quay.io
+```
+
+Next, use the following command to push your image to the registry:
+
+```
+podman push <image-ID> docker://quay.io/myorg/repo:tag
+```
+
+Replace <image-ID> and myorg/repo:tag with the actual ID received from the `podman build` command and repo:tag of your choice.
+
```

Comment:
It's not necessary to push to a registry for this process.

You can just pull directly from local container storage, which is one of the examples shown below:

```
```
# to pull from local storage
skopeo copy containers-storage:quay.io/myorg/myrepo:mytag oci-archive:my-custom-rhcos.ociarchive
```

Reply:
Yes, that makes sense.
Do you think I can change this to something like:


> You will want to push this image to a registry so that it can be used as an
`osImageUrl` in a MachineConfig as documented in the OpenShift docs.

If that's the case, start by logging into your account with Podman:

```
podman login quay.io
```

Next, use the following command to push your image to the registry:

```
podman push <image-ID> docker://quay.io/myorg/repo:tag
```

Replace <image-ID> and myorg/repo:tag with the actual ID received from the `podman build` command and repo:tag of your choice.

Reply:
I don't think these docs are intended to be comprehensive. In an effort to give the reader just enough info (i.e. exactly what they need to get the job done and not much extra) they intentionally leave out some basic steps. If the reader is updating an OS Image in an OpenShift cluster (an advanced operation) then we can probably assume they know how to get their image into a registry.

Reply:
@c4rt0 - I think maybe we can achieve both goals here. goals:

1. If the reader knows how to push then they don't need to be bothered with more text on how to do it.
2. If the reader doesn't know how to push we also make it easy for them to access that information.

How about we update this PR to link the `push` word in the earlier sentence about `OSImageURL` to https://docs.podman.io/en/latest/markdown/podman-push.1.html ?

Reply:
Agreed. Thank you for the suggestions.